src/Quartz/App_Packages/LibLog.4.2/LibLog.cs

```diff
@@ -492,7 +492,7 @@ static ILog For<T>()
             return GetLogger(typeof(T));
         }
 
-#if !LIBLOG_PORTABLE
+#if STACKTRACE && !LIBLOG_PORTABLE
```

Comment:
I was actually able to revert these changes because LibLog portable works directly on .NET Core now.